pyproject.toml

```diff
@@ -138,7 +138,7 @@ readme-renderer = { version = "*", optional = true }  # groups = ["dev"]
 
 
 [tool.poetry.extras]
-user = ["tqdm", "attrs", "watchdog", "keyring", "packaging", "PyYAML", "requests", "matplotlib", "scipy", "Pillow", "imageio", "scikit-image", "pymssql", "natsort", "tomlkit", "pynetdicom", "pylibjpeg-libjpeg", "dbfread", "xmltodict", "timeago", "xlsxwriter", "plotly", "dicompyler-core", "numpy", "pandas", "pydicom", "shapely", "streamlit", "streamlit-ace", "streamlit-analytics", "pylinac", "scikit-learn"]
+user = ["tqdm", "attrs", "watchdog", "keyring", "packaging", "PyYAML", "requests", "matplotlib", "scipy", "Pillow", "imageio", "scikit-image", "pymssql", "natsort", "tomlkit", "pynetdicom", "pylibjpeg-libjpeg", "dbfread", "xmltodict", "timeago", "xlsxwriter", "plotly", "dicompyler-core", "numpy", "pandas", "pydicom", "shapely", "streamlit", "streamlit-ace", "streamlit-analytics", "pylinac", "scikit-learn", "sqlalchemy"]
```

Comment:
This part of the file is autogenerated. Instead you'll want to add:

```toml
sqlalchemy = { version = "*", optional = true }  # groups = ["user", "dev"]
```

above, and then run:

```
poetry run pymedphys dev propagate
```

Reply:
You'll then need to add sqlalchemy to the following file:

https://github.com/pymedphys/pymedphys/blob/main/lib/pymedphys/_imports/imports.py

Then you should be able to import it with:

```python
from pymedphys._imports import sqlalchemy
```

This is to support optional imports within PyMedPhys. All external imports of PyMedPhys are "lazy loaded", meaning, if the user doesn't have that package installed they can still use the parts of PyMedPhys that don't need it.